src/DI/PhpReflection.php

```diff
@@ -81,6 +87,9 @@ public static function expandClassName($name, \ReflectionClass $rc)
 		if (empty($name)) {
 			throw new Nette\InvalidArgumentException('Class name must not be empty.');
 
+		} elseif (in_array($name, PHP_VERSION_ID >= 70000 ? ['string', 'int', 'float', 'bool', 'array', 'callable'] : ['array', 'callable'], TRUE)) {
```

Comment:
It is needed. Naming class `string` is valid in PHP < 7.0 and it needs to be expanded if it's namespaced. Should I add a test for this, so the check isn't removed accidentaly in the future?